### PR TITLE
Dev: Convert to fully qualified collection names

### PR DESCRIPTION
**Pull request by ccamacho**
Starting on Ansible 2.9 all tasks can use
the fully defined collection name the builtin
runtime is specified in:
https://github.com/ansible/ansible/blob/devel/lib/ansible/config/ansible_builtin_runtime.yml

### CODE REVIEW

**Comment by ccamacho** [comment]
This is passing the functional tests but be aware that it breaks the workload migrations.

**Comment by ccamacho** [comment]
Passing all, the breaking change was not related to this PR.

**Review by jistr — CHANGES_REQUESTED** [review]

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,7 @@ install: os_migrate-os_migrate-latest.tar.gz
 		echo "Sourcing venv."; \
 		source /root/venv/bin/activate; \
 	fi; \
+	ansible-galaxy collection install -r collection-requirements.yml; \
```
[the remark this answers is not in the document]
Comment (ccamacho) [reply Makefile]: moved the req files to the galaxy file, commit message updated, merged the pending PRs

**Review by jistr — APPROVED** [review]
Thanks for the updates, LGTM. The last bugfix only edits a .py file constant so it will not conflict, we can merge in any order.

**Comment by ccamacho** [comment]
Let's re-execute the e2e tests to be sure there are no NITs in the rebase

**Comment by jistr** [comment]
Yes +1

**Comment by jistr** [comment]
Something's wrong with OSP 10 cloud it seems, i'll check tomorrow.

**Comment by jistr** [comment]
The CI on my patch failed too.

**Pull request merged**